src/styles/article/our_channels.scss

```diff
@@ -0,0 +1,18 @@
+.container3 {
+    margin-top: 15px;
+    margin-bottom: 30px;
+
+    .title {
+        font-weight: bold
+    }
+
+    ul {
+        list-style: none;
+        padding: 0;
+    }
+
+    .call-to-action {
+        font-weight: bold;
+        font-size: 18px;
+    }
```

Comment:
Jako, że jest to jedyna klasa która "zostanie", przenieś ją do article.scss i nazwij .call-to-join-channels